msdm/algorithms/lrtdp.py

```diff
@@ -50,16 +51,14 @@ def plan_on(self, mdp: MarkovDecisionProcess):
         res = self.res
         res.policy = AssignmentMap()
         res.Q = AssignmentMap()
-        for s in mdp.state_list:
+        for s in sum(self.res.trials, []):
```

Comment:
might make sense to take unique states here instead? Might need to make this reachable states from start state, since things not along the final path might not be things we want to commit to in the partial policy.